packages/xo-web/src/common/xo/index.js

```diff
@@ -1778,6 +1778,38 @@ export const setVif = (
     rateLimit,
   })
 
+export const addAclRule = ({
```

Comment:
ACL is a generic term, which is already used in XO, make sure your names are specific enough.